# Defines the desired trajectory over a planning horizon.
# HORIZON = 41

auv_control/TrajectoryPoint[41] trajectory
